formula-android/src/main/java/com/instacart/formula/android/internal/FeatureBinding.kt

```diff
@@ -1,67 +1,39 @@
 package com.instacart.formula.android.internal
 
-import com.instacart.formula.Action
-import com.instacart.formula.Evaluation
-import com.instacart.formula.Formula
-import com.instacart.formula.FormulaContext
-import com.instacart.formula.Snapshot
 import com.instacart.formula.android.FeatureFactory
 import com.instacart.formula.android.FragmentKey
 import com.instacart.formula.android.FeatureEvent
+import com.instacart.formula.android.FragmentEnvironment
+import com.instacart.formula.android.FragmentId
 
 /**
- * Defines how a specific key should be bound to its [FeatureFactory],
+ * Defines how a specific key should be bound to its [FeatureFactory]
  */
-internal class FeatureBinding<in Component, in Dependencies, in Key : FragmentKey>(
-    private val type: Class<Key>,
-    private val feature: FeatureFactory<Dependencies, Key>,
-    private val toDependencies: (Component) -> Dependencies
-) : Binding<Component>() {
-
-    private val formula = object : Formula<Input<Component>, Unit, Unit>() {
```

Comment:
Simplified this logic by removing `Formula` logic and keeping only `createFeature`